app/views/leaderboard.scala.html

```diff
@@ -8,7 +8,7 @@
 @leaderboardStats = @{UserStatTable.getLeaderboardStats(10)}
 @leaderboardStatsThisWeek = @{UserStatTable.getLeaderboardStats(10, "weekly")}
 @leaderboardStatsByOrg = @{UserStatTable.getLeaderboardStats(10, "overall", true)}
-@leaderboardStatsOrg = @{UserStatTable.getLeaderboardStats(10, "overall", false, UserOrgTable.getAllOrgs(user.get.userId).headOption)}
+@leaderboardStatsOrg = @{UserStatTable.getLeaderboardStats(10, "overall", false, UserOrgTable.getOrg(user.get.userId))}
```

Comment:
Since you made the change in `userProfile.scala.html` to replace the repeated uses of `UserOrgTable.getOrg()` and `OrganizationTable.getOrganizationName()` with a saved one at the top, could you do that for this file too?? Might as well improve page performance when we see small stuff like this!